### PR TITLE
Hamburger-menu-feature

### PR DESCRIPTION
**Pull request by kayjeee**
Fixed Hamburger button and created a new branch for the hamburger menu and its links

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+      <ul class="navigation-bar nav-menu">
+        <li><a href="#portfolio" class="list-item">Portfolio</a></li>
+        <li><a href="#about" class="list-item">About</a></li>
+        <li><a href="#contact" class="list-item">Contact</a></li>
+      </ul>
+    </nav>
+    <div class="hamburger">
+      <i class="menu-icon fa-solid fa-bars"></i>
+      <i class="cross-icon fa-solid fa-times"></i>
+    </div>
```
Comment (abdulhamiid) [inline index.html:32]: You did a good job working on implementing the changes suggested by the previous reviewers. 👍 💯 

**_[OPTIONAL]_** - I could see that the hamburger is placed outside the `nav` tag. Please kindly consider wrapping the hamburger button inside the `nav` tag. The nav tag is for semantics reasons and user experience for mobile devices. 💪

**Review by abdulhamiid — APPROVED** [review]
# :green_circle:  Approved! :partying_face:
Hi @kayjeee 👋, 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (kayjeee) [reply index.html:32]: @abdulhamiid  thanks buddy I will do that, great looking out. I started following you on GitHub . keep up the good work

**Comment by kayjeee** [comment]
Code review approved now we merge with the main repository
our mobiile first feature

**Pull request merged**